core/templates/dev/head/services/AutoplayedVideosServiceSpec.js

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+describe('AutoplayedVideosService', function() {
+  var AutoplayedVideosService;
+
+  beforeEach(module('oppia'));
+  beforeEach(inject(function($injector){
+    AutoplayedVideosService = $injector.get('AutoplayedVideosService');
+  }));
+
+  it('should add video to a list of autoplayed videos', function(){
+    AutoplayedVideosService.addAutoplayedVideo('abcdef');
```

Comment:
Have you checked that `videoId` will be of string type? (Just asking, because I haven't checked it yet)

Reply:
Yes, it's a string. It is mentioned in `assets/rich_text_components_definitions.js`
``` "customization_arg_specs": [{
      "name": "video_id",
      "description": "The YouTube id for this video. This is the 11-character string after \"v=\" in the video URL.",
      "schema": {
        "type": "unicode"
      } ```

Reply:
Cool, let's make it 11-character long unicode type string then, for example `Ntcw0H0hwPU`.
While testing we like to assign variables to values that are close to what they'll get assigned to in real :)

Reply:
@DubeySandeep my bad. Made the changes.
Thanks!